build.gradle.kts

```diff
@@ -17,7 +17,7 @@
  */
 
 plugins {
-    id("com.android.application") version "7.4.2" apply false
-    id("com.android.library") version "7.4.2" apply false
-    id("org.jetbrains.kotlin.android") version "1.8.0" apply false
-}
\ No newline at end of file
+    id("com.android.application") version "8.0.0" apply false
+    id("com.android.library") version "8.0.0" apply false
```

Comment:
AGP (Android Gradle Plugin) 8.0.0 is only compatible with the latest version of Android Studio (released this month), Flamingo. What do you think about including this information in the README?

Then I will study carefully what changes with this version:
https://developer.android.com/build/releases/gradle-plugin#8-0-0

Reply:
Much of the projects I'm contributing to are already using latest versions. For me it is normal that latest version of Android is beeing used, but I can add info in the README if you want.

Reply:
Just to make it more informative, in the instructions on how to build inform the version of Android Studio.

Reply:
Rebased and added changes in the readme.